OMCompiler/Compiler/NFFrontEnd/NFInst.mo

```diff
@@ -2871,7 +2871,7 @@ algorithm
       then
         Sections.join(Sections.new(eq, ieq, alg, ialg), sections);
 
-  end match;
+  end matchcontinue;
```

Comment:
Why `matchcontinue`? You could make it an if-statement to check in the case below instead. `matchcontinue` is bad, bad, bad

Reply:
Also, MULTIPLE_SECTIONS_IN_FUNCTION is still an error, but you will succeed anyway? You really need to put this on a flag and make it a warning if the flag is given. You should never get errors if you succeed, and you should not succeed if the model is invalid (unless you give some flag saying to allow this explicitly).

Reply:
I'm also not a fan of allowing this at all since they're going to fix it in the library anyway, and as we all know there's nothing more permanent than a temporary solution.

Reply:
This is a draft. I will remove that when they fix TILMedia, they are working on it. I just wanted to not die there.

Reply:
If it was fixed properly, it could be merged into master. Nothing really problematic about allowing this on a flag, right? (If the error-messages gives the name of the flag it's even rather user-friendly)

Reply:
First priority: get this model running. Second priority: fix this PR draft so we can push it in :)

We need a flag -d=permisive (any other suggestions?) where we allow:
- enumerations as integer and vice-versa
- this thing
- other annoying D* things we encounter in the libraries

Reply:
I think you need to have one debug-flag for each of the annoying things, like gcc does, e.g. `-fdollars-in-identifiers`. With no convenient way to enable all.